create-snowpack-app/app-template-minimal/snowpack.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
ah I love this! Either in this PR or a follow up one, can you also go through the other template config files and add this format for any properties not defined?

Reply:
yeah for sure, also maybe adding readmes to them all?

Reply:
they should all have READMEs! feel free to tackle that in seperate PR